lib/configuration/connection_configuration.js

```diff
@@ -32,35 +32,45 @@ function readTokenFromFile(fixedConfiguration) {
   validateOnlyUserReadWritePermission(resolvedPath);
   fixedConfiguration.token = fs.readFileSync(resolvedPath, 'utf-8').trim();
   if (!fixedConfiguration.token) {
+    Logger.getInstance().error('The token does not exist or has empty value.');
     throw new Error('The token does not exist or has empty value');
   }
   const tokenChecksum = generateChecksum(fixedConfiguration.token);
   Logger.getInstance().info('Token used in connection has been read from file: %s. Checksum: %s', resolvedPath, tokenChecksum);
 }
 
 function loadConnectionConfiguration() {
+  Logger.getInstance().trace('Loading connection configuration from the local files.');
```

Comment:
Perhaps adding ellipsis at the end might suggest that the process is ongoing?

Reply:
Nice idea, added 👍